Simplenote/RecoveryUnarchiver.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+import UniformTypeIdentifiers
+import CoreData
+
+public class RecoveryUnarchiver {
+    private let fileManager: FileManager
+
+    public init(fileManager: FileManager = .default) {
+        self.fileManager = fileManager
+    }
+
+    var simperium: Simperium {
+        SPAppDelegate.shared().simperium
```

Comment:
Same note as in mac, since the AppDelegate will use this, and has direct access to Simperium, this class can get the reference via initializer